appveyor.yml

```diff
@@ -40,8 +40,8 @@ branches:
   - /^(?i:devprop).*$/
 
 init:
-  # Note: python 2.7 32bit is already on the path. We want v2.7 64bit,
-  # so we must add v2.7 64bit earlier on the PATH so that CMake finds it when
+  # Note: python 3.7 32bit is already on the path. We want v3.7 64bit,
+  # so we must add v3.7 64bit earlier on the PATH so that CMake finds it when
```

Comment:
I don't know if this statement is true. You could just remove the comment or say "Avoid using other version of python that might already be on the PATH."

Reply:
Done